### PR TITLE
Add support for probot/stale

### PR DESCRIPTION
**Pull request by mangelajo**

### CODE REVIEW

**Comment by mangelajo** [comment]
Discussed upstream, agreement that it was a good thing ^ :-) please review / comment / approve. Thanks :)

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#   Your comment here.
+
+# Limit the number of actions per hour, from 1-30. Default is 30
+limitPerRun: 30
+
+# Limit to only `issues` or `pulls`
+# only: issues
+
+pulls:
+   daysUntilStale: 30
```
Comment (sridhargaddam) [inline .github/stale.yml:52]: In Line3, it says "... or **Pull Request** becomes stale". So, does this (aka Line52) override the value in Line4?

nit: If so, do we want to remove "... Pull Request ..." from Line3?

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
```
Comment (mangelajo) [inline .github/stale.yml:3]:
```suggestion
# Number of days of inactivity before an Issue becomes stale (pull request is overridden later) 
```

**Comment by mangelajo** [comment]
Comments addressed, I'm merging and activating.

**Pull request merged**